Makefile

```diff
@@ -26,8 +26,10 @@ format:
 .PHONY: compile
 compile: format
 	@poetry check
-	@poetry run ruff poetry_docker_plugin
-	@poetry run mypy poetry_docker_plugin
+	@poetry run ruff check --diff --no-fix poetry_docker_plugin tests
```

Comment:
added the `--diff` and `--no-fix` with the intention of using pre-commit to fix before commit and the github workflow to be a quality gate.